### ISSUE TITLE
Test failure caused by regression in node.js path module in 5.7.0

### ISSUE THREAD

**Issue by es128** [paulmillr/chokidar#442]
https://github.com/nodejs/node/pull/5389

```
λ nvm use 5
Now using node v5.7.0

λ node
> path.relative('/tmp/1/2/3', '/tmp/1/2')
'../../2'

λ nvm use 5.6
Now using node v5.6.0

λ node
> path.relative('/tmp/1/2/3', '/tmp/1/2')
'..'
```

```
  1) chokidar fs.watchFile (polling) getWatched should set keys relative to cwd & include added paths:

      Uncaught AssertionError: expected { Object (../../test-fixtures, ., ...) } to deeply equal { Object (., .., ...) }
      + expected - actual

           "change.txt"
           "subdir"
           "unlink.txt"
         ]
      -  "../../test-fixtures": [
      +  "..": [
           "1"
         ]
         "subdir": []
       }

      at Assertion.assertEqual (node_modules/chai/lib/chai/core/assertions.js:485:19)
      at Assertion.ctx.(anonymous function) [as equal] (node_modules/chai/lib/chai/utils/addMethod.js:41:25)
      at FSWatcher.<anonymous> (test.js:1563:48)
```

**Comment by paulmillr** [comment]
they should fix it in 5.7.1 https://github.com/nodejs/node/pull/5464

**Comment by es128** [comment]
Yup, just wanted this open to track it until the release actually happens. Not a big deal though
